scripts/image_list.rb

```diff
@@ -277,6 +277,9 @@ def images
 # Iterate over all permutations, rendering the chart to obtain all possible
 # images.
 permutations.each do |permutation|
+  # Skip forbidden combination of routing_api and eirini
+  next if permutation['routing_api'] && permutation['eirini']
```

Comment:
This isn't working: we only run 2 permutations: all enabled, and all disabled. So you are skipping the "all enabled" option, and we will not record the images required for Eirini.

There is already a clause I added earlier where we set `install_stacks` to just `sle15`. Add a line there to force `routing_api` to disabled...

If you are feeling generous, maybe move the settings below for embedded and external database into the same location (but outside the Eirini check) too, so all the special cases are close to each other in the code.

Reply:
> we only run 2 permutations

https://github.com/cloudfoundry-incubator/kubecf/blob/master/scripts/image_list.rb#L270-L277

Reply:
Fixed in commit 6744838b5442e8280e922c904b1038da0cabe333. Rebased to get image_list.rb changed. Fixed. Collapsed history, force pushed.